gradle/instrumentation.gradle

```diff
@@ -58,6 +58,17 @@ afterEvaluate {
 
     testImplementation deps.testcontainers
 
+    // Include instrumentations instrumenting core JDK classes tp ensure interoperability with other instrumentation
+    testInstrumentation project(':instrumentation:executors:javaagent')
```

Comment:
Moved here for `mergeServiceFiles()`

Reply:
Looks like this approach won't be feasible - it really does explode things too much to do this all in each project. With this build, the build cache (which the shadowJar is a part of) gets too large to upload. I think I'll punt on this PR in favor of @iNikem's volunteered work of the properly isolated extensions jar :) So I'll also punt on https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2634#discussion_r603104376 until then I think as I can't test behavior with / without an override present since it's always in the bootstrap for our tests right now.